File: LeetCode/LeetCode Practice Problems/1347- Minimum Number of Steps to Make Two Strings Anagram.cpp
class Solution {
public:
    int minSteps(string s, string t) {
        map<char,int> m;
        for(char c : s)
            m[c]++;
        
        for(char c : t)
            if(m[c] > 0)
                m[c]--;
        
        int ans = 0;
        for(auto i : m)
            ans+=i.second;
        
        return ans;
    }
};

File: LeetCode/LeetCode Practice Problems/1347-Minimum Number of Steps to Make Two Strings Anagram.java
class Solution {
    public int minSteps(String s, String t) {
        int cnt[] = new int[26];
        
        for(char c: s.toCharArray()) {
            cnt[c - 'a']++;
        }
        
        for(char c: t.toCharArray()) {
            if(cnt[c - 'a'] > 0) {
                cnt[c - 'a']--;
            }
        }
        
        int ans = 0;
        for(int c: cnt) {
            ans += c;
        }
        
        return ans;
    }
}
